build.gradle

```diff
@@ -27,7 +27,7 @@ configurations {
 
 repositories {
     maven {
-        url "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+        url "https://s01.oss.sonatype.org/content/repositories/releases/com/techsophy/multitenancy-mongo/0.2.0/"
```

Comment:
no, its a release version dont need to change this

Reply:
Yes vaibhav reverting back to old one only